conformance/04_management_test.go

```diff
@@ -131,12 +131,17 @@ var test04ContentManagement = func() {
 				req := client.NewRequest(reggie.GET, "/v2/<name>/tags/list")
 				resp, err := client.Do(req)
 				Expect(err).To(BeNil())
-				Expect(resp.StatusCode()).To(Equal(http.StatusOK))
-				tagList := &TagList{}
-				jsonData := []byte(resp.String())
-				err = json.Unmarshal(jsonData, tagList)
-				Expect(err).To(BeNil())
-				Expect(len(tagList.Tags)).To(BeNumerically("<", numTags))
+				Expect(resp.StatusCode()).To(SatisfyAny(
+					Equal(http.StatusNotFound),
+					Equal(http.StatusOK),
+				))
+				if resp.StatusCode() == http.StatusOK {
+					tagList := &TagList{}
+					jsonData := []byte(resp.String())
+					err = json.Unmarshal(jsonData, tagList)
+					Expect(err).To(BeNil())
+					Expect(len(tagList.Tags)).To(BeNumerically("<", numTags))
+				}
```

Comment:
Since we are pushing and deleting a single tag, does it make sense to include a strict equal comparison?

```suggestion
				expectTags := numTags - 1
				if resp.StatusCode() == http.StatusOK {
					tagList := &TagList{}
					jsonData := []byte(resp.String())
					err = json.Unmarshal(jsonData, tagList)
					Expect(err).To(BeNil())
					Expect(len(tagList.Tags)).To(Equal(expectTags))
				}
				if resp.StatusCode() == http.StatusNotFound {
					Expect(expectTags).To(Equal(0))
				}
```

Reply:
Could we maybe push 2 tags to avoid deletion of the repo? 
if not then as long as we accept 404 I think It’s ok.

Reply:
I'm okay with either, but if we added a second tag, we'd need to remove it in a later teardown.

Reply:
@sudo-bmitch I've applied the requested changes.